src/Config/GlobalSettings.cs

```diff
@@ -29,23 +29,27 @@ public record ApiSettings(
     /// <summary>
     /// Holds the global settings used at runtime for REST Apis.
     /// </summary>
+    /// <param name="Enabled">If the REST APIs are enabled.</param>
     /// <param name="Path">The URL prefix path at which endpoints
     /// for all entities will be exposed.</param>
     public record RestGlobalSettings(
+        bool Enabled = true,
```

Comment:
```suggestion
        bool isEnabled= true,
```

Reply:
changing this would require changing at a lot of places, it was previously named `Enabled` only. If you look at the `ApiSettings` class

Reply:
maybe not a part of this PR, if this is needed.